DEPS

```diff
@@ -18,7 +18,7 @@ deps = {
     Var('chromium_git') + '/external/gflags/src@e7390f9185c75f8d902c05ed7d20bb94eb914d0c', # from svn revision 82
 
   'src/third_party/junit-jar':
-    Var('chromium_git') + '/external/webrtc/deps/third_party/junit@f35596b476aa6e62fd3b3857b9942ddcd13ce35e', # from svn revision 3367
+    Var('chromium_git') + '/external/junit@9e98a85ecf6e857523fcda9150756297a64bccba', # from svn revision 3367
```

Comment:
👍 on the change, but 👎 on keeping the comment : this is advancing the reference to the latest and greatest (which should be fine -- we don't really rely on junit for anything we're using client-side, this seems like it's more to satisfy build stuff), not to some historic 'svn revision 3367' which doesn't even seem to exist in this history.